core/src/test/java/greencity/service/impl/LanguageServiceImplTest.java

```diff
@@ -1,18 +1,22 @@
 package greencity.service.impl;
 
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.when;
-
+import greencity.ModelUtils;
 import greencity.constant.AppConstant;
 import greencity.dto.language.LanguageDTO;
+import greencity.entity.Language;
+import greencity.exception.exceptions.LanguageNotFoundException;
 import greencity.repository.LanguageRepo;
 import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
 import javax.servlet.http.HttpServletRequest;
+import static org.junit.Assert.assertEquals;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import static org.mockito.ArgumentMatchers.anyString;
```

Comment:
You should follow the same style as others. Pleas ask your teammates for code style configuration.